processing-service-core/api/swagger/swagger.yaml

```diff
@@ -38,6 +38,8 @@ tags:
     description: Operations for Conditions on Processing Rules.
   - name: Workflows
     description: Workflow operations.
+  - name: Global Config
+    description: Global Configuration.
```

Comment:
In my earlier comment I meant that the name should be 'Global Configuration' not the shortened version. The description should change to "Global Configuration operations.".